src/org/zaproxy/zap/extension/bugtracker/ZapAddOn.xml

```diff
@@ -0,0 +1,17 @@
+<zapaddon>
+	<name>Bug Tracker</name>
+	<version>1</version>
+	<status>alpha</status>
+	<description>Bug Tracker extension.</description>
+	<author>ZAP Dev Team</author>
+	<url></url>
+	<changes>First version</changes>
+	<extensions>
+		<extension>org.zaproxy.zap.extension.bugtracker.ExtensionBugTracker</extension>
+	</extensions>
+	<ascanrules/>
+	<pscanrules/>
+	<filters/>
+	<not-before-version>2.6.0</not-before-version>
```

Comment:
Missing new class (PopupMenuAlertItem), we can target 2.5.0 but we need to use reflection.